var/spack/repos/builtin/packages/mariadb/package.py

```diff
@@ -21,6 +21,7 @@ class Mariadb(CMakePackage):
     homepage = "https://mariadb.org/about/"
     url = "http://ftp.hosteurope.de/mirror/archive.mariadb.org/mariadb-10.2.8/source/mariadb-10.2.8.tar.gz"
 
+    version("11.1.1", sha256="272051c65cc46bace572eb01e036d3601c2d1e83af6ca84a03ddb3c3860e0385")
```

Comment:
According to https://mariadb.com/kb/en/mariadb-11-1-1-release-notes/, 11.1.1 is a release candidate. Does one of the stable versions (e.g., 10.11.4) also build with GCC 13?

Reply:
Yes, I saw it was a RC. I was/am getting 403 for the other more recent releases from that mirror - sorry, should have said that's why I'd skipped.
http://ftp.hosteurope.de/mirror/archive.mariadb.org/mariadb-10.11.4/ http://ftp.hosteurope.de/mirror/archive.mariadb.org/mariadb-11.0.2/

Can wait for a proper release if that's the preference.

Reply:
Weird, that seems to be limited to that mirror, though. https://mirror.wtnet.de/mariadb/mariadb-11.0.2/ works for me.

Reply:
Updated to use that mirror location for `@11.0.2` so that we aren't using the RC.